RELEASING.md

```diff
@@ -44,8 +44,23 @@ that builds+pushes the engine image to our registry with a tag matching
 At the end of this workflow, a new PR will automatically be created to bump the
 Engine version in the various SDKs.
 
+⚠️ **There is currently [an issue](https://github.com/dagger/dagger/issues/4050) where workflows do not automatically run in this PR.**
+Instead, you can trigger them to run by pushing an empty commit and then reverting it, e.g.
+
+```console
+gh pr checkout -f <PR number>
+git commit --allow-empty -m "Empty-Commit"
+git push
+git reset --hard HEAD^
```

Comment:
Technically you could also trigger it with `git commit  --amend --no-edit` so you do not have an extra commit :)